### ISSUE TITLE
Lists where the prefix already forms part of the code

### ISSUE THREAD

**Issue by BobHarper1** [org-id/register#75]
This arose in a discussion here https://github.com/TechforgoodCAST/find-that-charity/issues/13

As an example, the Scottish Charity Commission search facility returns identifiers with 'SC' which is their own prefix to each code. As org-id suggests adding the prefix `GB-SC-` I think some confusion may arise with users, do they therefore choose to use `GB-SC-SC012345` or `GB-SC-012345`?

**Comment by ScatteredInk** [comment]
I think they want `GB-SC-SC012345` because the pattern should be `pointer to the register`-`registered entity identifier`. 

It might in future be preferable to keep orgID prefixes distinct from prefixes used in the registers themselves - where these are known - to avoid this kind of confusion.

In the longer term, orgID could be used to look up whether prefixes are semantically-meaningful. There is a possible use-case in, e.g. devolved systems like the UK, where you know an identifier comes from a country but not necessarily what register it lives in.

**Comment by BobHarper1** [comment]
It's more of a signposting issue, making clear to users that the `SC` (in this case) that appears in the code is not part of the org-id prefix. Perhaps overcome with how we present examples.

**Comment by timgdavies** [comment]
Agreed: this is a signposting issue and we should try and bring examples into the interface more to help people get this right.
